### ISSUE TITLE
Nuxt Security Documentation site layout styling broken

### ISSUE THREAD

**Issue by jmvanderpol** [Baroshem/nuxt-security#557]
When navigating to the Nuxt Security documentation both Opera and Safari show the web pages without correct layout positioning. Yesterday the pages were correct.

This is what Opera shows today for the documentation site:
<img width="878" alt="Screenshot 2024-11-14 at 1 41 52 PM" src="https://github.com/user-attachments/assets/ba607eda-dd41-4279-9ced-3832c4cf1da5">

**Comment by Baroshem** [comment]
Hey there!

It seems that it is broken in other browsers as well. I just checked Brave and it is the same.

Maybe it has to do with the Node version bump from 18.0 to 20.0?

**Comment by Baroshem** [comment]
The documentation should look good now.

There is an issue in the documentation of the project that was caused by bumping the Node version, and clearing cache for depencendies which resulted in downloading newest packages where `pinceu` and `docus` are causing problems.

I did a rollback to a version from 11 days ago @vejja 

As this is docs setup is problematic, I think it is just a good time to migrate to newest versions of nuxt documenation for modules like it is for Nuxt Image/Content for example. 

I will work on it in the upcoming days. @vejja let's not push anything to main until I do so because otherwise the current code will be taken that basically breaks the whole UI of docs.
